app/jobs/make_mine_job.rb

```diff
@@ -0,0 +1,22 @@
+
+class MakeMineJob < ApplicationJob
+  queue_as :default
+
+  def perform(width, height, board, amount = 1)
```

Comment:
Its not required to pass in 1, performance increases can be achieved by changing this